### PR TITLE
Update spec file

### PR DESCRIPTION
**Pull request by tyll**
This change is [Reviewable](https://reviewable.io/reviews/nmstate/nmstate/575)

### CODE REVIEW

**Comment by tyll** [comment]
This needs the CI to not use Fedora 29 anymore because NM is too old there for the libnm dependency update.

**Comment by EdDev** [comment]
On what PR this one is dependent on?

**Comment by tyll** [comment]
> On what PR this one is dependent on?

#543 or #580

File: packaging/nmstate.py3.spec [hunk 1]
```diff
@@ -24,10 +24,12 @@ provider support on the southbound.
 %package -n python3-%{libname}
 Summary:        nmstate Python 3 API library
 Requires:       python3-dbus
-Requires:       NetworkManager-libnm >= 1:1.12
+Requires:       NetworkManager-libnm >= 1:1.20
 # Use Recommends for NetworkManager because only access to NM DBus is required,
 # but NM could be running on a different host
 Recommends:     NetworkManager
```
Comment (EdDev) [inline packaging/nmstate.py3.spec]: Should we request NM 1.20 (like we do with libnm)?

Comment (tyll) [reply packaging/nmstate.py3.spec]: NM has a versioned dependency on libnm, therefore libnm >= 1.20 implies that NM also needs to be updated (if it is installed).

**Pull request merged**